src/main/java/BobBot.java

```diff
@@ -0,0 +1,93 @@
+import java.util.Scanner;
+
+public class BobBot {
+    
+    private static Task[] allTasks = new Task[100];
```

Comment:
Maybe can replace 100 as a static final int declaration at the top as MAX_INT? This is to avoid "magic numbers"
```suggestion
    private static Task[] allTasks = new Task[MAX_INT];
```

Reply:
Thanks for pointing out!